src/utils.js

```diff
@@ -0,0 +1,9 @@
+function findHighest(a,b,c) {
```

Comment:
- Let's add some tests for the function. You will need to create a `tests` directory in the project directory. 
Inside the tests directory, let's create a `utils.js` file that will contain tests for the utils file.

Then, let's add tests for the `findHighest` function using mocha. 

We should also have an npm script - `test` to run the tests

See https://mochajs.org/ for some guide

Reply:
ok